### DISCUSSION TITLE
Use a custom repo

### DISCUSSION THREAD

**Discussion by zerr0s** [artifacthub/hub#3947] [Q&A] [answered]
Hello,

I have few custom registries for docker/helm. these registries expose a self-signed certificate.
When I add these registries I get "certificated signed by unknown authority".

Is there a way to provide a custom ca to the hub ?

Thanks.

**Comment by tegioz** [comment]
Hi @zerr0s

No, at the moment we don't provide any mechanism to add custom certificate authorities.

**Comment by zerr0s** [comment]
Ok I'll try to import all ca into /etc/ssl/certs. Hope that will solve my issue.

**Comment by zerr0s** [comment]
Hello,

I can't understand why the hub doesn't work after I add all CA to /etc/ssl/certs directory.

I have built a new image with all ca added, update the bundle with update-ca-certificates, and finally I have tested the connection with curl. All is working well in the container.

When the hub tries to connect, i still have a SSL handshake failure. It seems that the hub is not using the system trustore.

The hub should use the system trustore. But it doesn't seem to work. an idea please ?

**Comment by zerr0s** [comment]
I think that the problem is the tracker. I'll try to make the same config in tracker image.

**Reply by zerr0s** [reply, marked as answer]
adding the ca to the tracker image fix my problem
